source/content.css

```diff
@@ -879,6 +878,10 @@ body > .footer li a {
 	color: var(--github-gray-text) !important;
 }
 
+.dashboard p {
+	margin: 0 0 4px !important;
+}
```

Comment:
Unrelated, way to broad, not `!important`. Can be dropped

Reply:
Because after hiding some elements, the remaining elements are not aligned.

Before:
![image](https://user-images.githubusercontent.com/12368291/37253499-6e01ccca-2558-11e8-96ca-5d4e0d661982.png)

After:
![image](https://user-images.githubusercontent.com/12368291/37253489-594c6704-2558-11e8-8f8e-77abc1a165ee.png)

-----------------------

And the `!important` is required as it overrites the `!important` of `.m*-*` classes.

Reply:
Not true, the only `p` you're targeting (and that needs the spacing to be reduced) doesn't have any classes. And the selector is still too broad because `.dashboard` is the entire page. This is is also affected:

![](https://user-images.githubusercontent.com/1402241/37256351-f782723c-258b-11e8-9d08-95ecbb62e5e6.gif)